test/worlds/contact.sdf

```diff
@@ -8,6 +8,9 @@
     <plugin
       filename="ignition-gazebo-physics-system"
       name="ignition::gazebo::systems::Physics">
+      <contacts>
+        <include_entity_names>true</include_entity_names>
+      </contacts>
```

Comment:
Can we test the default behavior as well, i.e., without the `<contacts>` parameter?

Reply:
Added 3 different world files in the corresponding test cases :
* ``contact_with_entity_names.sdf`` : Has the parameter set to true
* ``contact_without_entity_names.sdf`` : Has the parameter set to false
* ``contact.sdf`` : Does not have the parameter (defaults to true)